chapters/remote/remote-pods.md

```diff
@@ -0,0 +1,42 @@
+# Remote Pods with JinaD
+
+### Prerequisites
+
+Before you start, make sure you have read the [prerequisites for using JinaD](https://docs.jina.ai/chapters/remote/jinad.html#prerequisites).
+
+## Inside a Flow
+
+A common case of using Jina remotely is to have a Flow running locally with Pod on the remote. Following code creates a Flow with a remote Pod on `1.2.3.4` and uploads the pod yml to remote using `upload_files`. All remote logs are collected by the Flow.
```

Comment:
maybe 

> on the remote

is a common phrase to use I just don't know. If its not then we could use 

'on a remote machine' or 
'with a Pod running remotely'

Reply:
or 'remote Pod'